src/main/java/org/minions/devfund/yuri/Squarelotron.java

```diff
@@ -0,0 +1,197 @@
+package org.minions.devfund.yuri;
+
+/**
+ * Represents the matrix with all functionality.
+ */
+public class Squarelotron {
+  public static final int COMPLETE_TURN = 4;
+  private int[][] matrix;
+  private int size;
+
+  /**
+   * Constructor Method.
+   *
+   * @param size matrix size.
+   */
+  public Squarelotron(final int size) {
+    matrix = new int[size][size];
+    this.size = size;
+    fillSquarelotron(matrix);
+  }
+
+  /**
+   * Gets the matrix matrix.
+   *
+   * @return matrix matrix.
+   */
+  public int[][] getMatrix() {
+    return matrix.clone();
+  }
+
+  /**
+   * Gets the matrix size.
+   *
+   * @return size.
+   */
+  public int getSize() {
```

Comment:
This method is not necessary.
Remove it!

Reply:
done